### PR TITLE
added dto tests review dto parser and tests

### PR DESCRIPTION
**Pull request by maks45**

### CODE REVIEW

File: src/main/java/com/maks/durov/amazonreviewapp/service/ReviewDtoParserService.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package com.maks.durov.amazonreviewapp.service;
+
+import com.maks.durov.amazonreviewapp.dto.ParsedReviewDto;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewDtoParserService {
+    private static final String SPLIT_REGEX = ",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)";
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/maks/durov/amazonreviewapp/service/ReviewDtoParserService.java:8]: if the last field of your review contains commas. Will you have the whole part of it or just the part before first comma?

File: src/main/java/com/maks/durov/amazonreviewapp/dto/ParsedReviewDto.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+public class ParsedReviewDto {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/maks/durov/amazonreviewapp/dto/ParsedReviewDto.java]:
```suggestion
public class ReviewDto {
```

File: src/main/java/com/maks/durov/amazonreviewapp/service/ReviewDtoParserService.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/com/maks/durov/amazonreviewapp/service/ReviewDtoParserService.java:8]: можеш відповісти в приватний чат

File: src/main/java/com/maks/durov/amazonreviewapp/dto/ParsedReviewDto.java (hunk 2 above)
Comment (maks45) [reply src/main/java/com/maks/durov/amazonreviewapp/dto/ParsedReviewDto.java]: done

File: src/main/java/com/maks/durov/amazonreviewapp/service/ReviewDtoParserService.java (hunk 1 above)
Comment (maks45) [reply src/main/java/com/maks/durov/amazonreviewapp/service/ReviewDtoParserService.java:8]: answered in private chat

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**